metaflow/plugins/test_unbounded_foreach_decorator.py

```diff
@@ -69,8 +69,12 @@ def control_task_step_func(self, flow, graph, retry_count):
         step_name = current.step_name
         control_task_id = current.task_id
         (_, split_step_name, split_task_id) = control_task_id.split('-')[1:]
-
-        executable = self.environment.executable(step_name)
+        # If we are running inside Conda, we use the base executable FIRST;
+        # the conda environment will then be used when runtime_step_cli is
+        # called. This is so that it can properly set up all the metaflow
+        # aliases needed.
```

Comment:
Which metaflow aliases are we interested in?

Reply:
The symlinks to metaflow and metaflow-custom.